src/solver/conv_MP_bidirectional_winograd.cpp

```diff
@@ -73,52 +76,43 @@ MIOPEN_DECLARE_ENV_VAR(MIOPEN_CONV_PRECISE_ROCBLAS_TIMING)
             solver::ConvMPBidirectWinograd<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>:: \
                 GetSolverWinoXformHWSize();
 
-#define DEFINE_SHADER_ALIASES(params)                      \
-    const auto group_cnt = (params).group_counts;          \
-    const auto& N        = (params).batch_sz;              \
-    const auto& K        = (params).n_outputs / group_cnt; \
-    const auto& C        = (params).n_inputs / group_cnt;  \
-    const auto& R        = (params).kernel_size_h;         \
-    const auto& S        = (params).kernel_size_w;         \
-    const auto& H        = (params).in_height;             \
-    const auto& W        = (params).in_width;              \
-    const auto& out_H    = (params).out_height;            \
+#define DEFINE_SHADER_ALIASES(params)                           \
+    const auto group_cnt = (params).group_counts;               \
+    const auto& N        = (params).batch_sz;                   \
+    const auto& K        = int((params).n_outputs / group_cnt); \
```

Comment:
[IMPORTANT] :warning:const reference to rvalue is an ___undefined behavior.___

Reply:
[IMPORTANT] This must be fixed in the original code.

Reply:
And if you want `int`, do not use C-style casts. Suggestion:
```suggestion
    const int K        = (params).n_outputs / group_cnt; \
```

Reply:
@pfultz2 perhaps we can add this as a check to cpp-check ?